gulpfile.js

```diff
@@ -1,27 +1,37 @@
-/******************************************************
- * Building the sass
-******************************************************/
-
-var gulp = require('gulp'),
-  path = require('path'),
-  sass = require('gulp-sass'),
-  config = require('./config.json');
+var gulp = require("gulp"),
+  path = require("path"),
+  sass = require("gulp-sass"),
+  config = require("./config.json"),
+  pug = require("gulp-pug");
 
 function resolvePath(pathInput) {
-  return path.resolve(pathInput).replace(/\\/g, '/');
+  return path.resolve(pathInput).replace(/\\/g, "/");
+}
+
+function paths() {
```

Comment:
Why do we need this function? It's returning a reference to an object that's already available in the same scope

Reply:
gulp.task ''build-sass'' requires paths() and resolvePath().
It was already written in the gulp file so i didn't removed this part of code.

I tried to remove unneeded code but it looks like the gulp task 'build-sass' needs this